pkg/controller/node/scheduler/taint_controller_test.go

```diff
@@ -457,7 +457,6 @@ func TestUpdateNodeWithMultiplePods(t *testing.T) {
 			pods: []v1.Pod{
 				*testutil.NewPod("pod1", "node1"),
 				*addToleration(testutil.NewPod("pod2", "node1"), 1, 1),
-				*addToleration(testutil.NewPod("pod3", "node1"), 1, -1),
```

Comment:
The test checks if the pod wasn't deleted.

Reply:
Oops, that is my mistake.
This PR should be dropped.
Thanks for your review, @gmarek